### PR TITLE
use random string for isolated build name

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by lcarva** [comment]
This looks good to me. Let's just fix up the unit tests.

**Comment by rcerven** [comment]
unit tests doesn't need fix, but fedora image can't be pulled

**Comment by lcarva** [comment]
Registry is going through maintenance, it appears: https://www.fedorastatus.org/q4maint.html

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -1421,8 +1421,7 @@ def render_name(self):
             if self.scratch:
                 name = 'scratch-{0}-{1}'.format(salt, timestamp)
             elif self.isolated:
-                name = wrap_name_from_git('isolated', timestamp, self.spec.git_uri.value,
-                                          self.spec.git_branch.value)
+                name = 'isolated-{0}-{1}'.format(salt, timestamp)
```
Comment (vrutkovs) [inline osbs/build/build_request.py]: A bit better version:
```
prefix = "scratch" if self.scratch else "isolated"
name = "-".join(prefix, salt, timestamp)
```

**Comment by rcerven** [comment]
release note updated

**Pull request merged**